mantidproject/mantid
# Add progress bar to SaveMDWorkspaceToVTK

Description of work.

This attaches MantidPlot's progress bar to VTK's internal progress callback, providing visual feedback to the user.

**To test:**



Load both an MDEventWorkspace and an MDHistoWorkspace and run SaveMDWorkspaceToVTK. Verify that progress from SaveMDWorkspaceToVTK is being reported.

[VTK merge request #2199](https://gitlab.kitware.com/vtk/vtk/merge_requests/2199) improves progress reporting from a vtkStructuredGrid.

There is no GitHub issue associated with this pull request

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
